src/main/scala/org/coroutines/AstCanonicalization.scala

```diff
@@ -130,10 +171,34 @@ trait AstCanonicalization[C <: Context] {
           (Nil, q"$method")
       }
       for (tpt <- tpts) disallowCoroutinesIn(tpt)
-      val (pdeclss, pidents) = paramss.map(_.map(canonicalize).unzip).unzip
+
```

Comment:
By `bynamess`, I meant `byNameParams`, yes.
`cs` is defined in my snippet, in the second line.